pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -1328,8 +1331,9 @@ cdef class PyObjectHashTable(HashTable):
             k = kh_put_pymap(self.table, <PyObject*>val, &ret)
             self.table.vals[k] = i
 
-    def lookup(self, ndarray[object] values) -> ndarray:
+    def lookup(self, ndarray[object] values, object mask = None) -> ndarray:
```

Comment:
can make the return type `ndarray[intp_t]` and remove the comment on the next line

Reply:
Done

Reply:
Correction: Cython complains about this, guess this is the reason for the comment